### DISCUSSION TITLE
Validator produces NDC errors however requirement not until 1/1/2025

### DISCUSSION THREAD

**Discussion by BBDURMC** [CMSgov/price-transparency-guide-validator#115] [Q&A] [answered]
The CSV validator is producing error results for NDC related fields however, the corresponding unit/measure data is not required until 1/1/25.
Please correct the validator.

Error reads: "If Code Type is NDC, then the corresponding drug unit of measure and drug type of measure data element must be encoded."

beverly_Dollinger@urmc.rochester.edu

**Comment by shaselton-usds** [comment, marked as answer]
Closing -- this discussion is for the HPT validator
